lib/wasmex/instance.ex

```diff
@@ -172,6 +172,47 @@ defmodule Wasmex.Instance do
   def memory(store, instance) do
     Wasmex.Memory.from_instance(store, instance)
   end
+
+  @doc ~S"""
+  Reads the value of an exported global.
```

Comment:
Do you want doctests here?

Reply:
I added some doctests

Reply:
These doctests are great! Thanks for adding them